frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-subscribe.tsx

```diff
@@ -267,11 +330,110 @@ export const OperatorHubSubscribeForm: React.FC<OperatorHubSubscribeFormProps> =
     );
   };
 
+  const isOpenshiftNamespace = (ns: string): boolean => {
+    return _.startsWith(ns, 'openshift-')
+  };
+
+  const createNamespaceAlert = () => {
+    const monitoring = <div style={{ marginLeft: '20px' }}>
+      <Checkbox
+        id={selectedTargetNamespace}
+        label="Enable operator recommended cluster monitoring on this namespace"
+        onChange={(val) => {
+          setEnableMonitoring(val)
+        }}
+        isChecked={enableMonitoring}
+        name="enableNamespaceMonitoring"
+      />
+      <small>Note: Enabling monitoring will allow any operator or workload running on this namespace to contribute metrics to the cluster metric set.</small>
```

Comment:
As mentioned in the other PR https://github.com/openshift/openshift-origin-design/pull/307#discussion_r363273005 we really want to warn the user that if they install a non redhat operator into an openshift namespace with enabled monitoring, that this voids user support. We should be clear on that, as that is the potential consequence of this action.

Reply:
I underline the severity of enabling monitoring in this case too. It voids user support as @LiliC mentioned.

Reply:
> I underline the severity of enabling monitoring in this case too. It voids user support

@LiliC @s-urbaniak @bparees in that case are we sure we want to give the user and way how to fiddle with the monitoring? Seeing all the comments (here and in the [design PR](https://github.com/openshift/openshift-origin-design/pull/307#pullrequestreview-338612452)) I wonder if we won't shoot ourselves into the leg with this single checkbox, since the variety of all the use-cases is just and potential damages is just too big. 
Thinking if it wont be for the best to leave it out for now and just aim for the suggested-namespace.

Thoughts ?

Reply:
> @LiliC @s-urbaniak @bparees in that case are we sure we want to give the user and way how to fiddle with the monitoring?

yes because we want to make the install story for redhat operators better than it currently is.  that's the primary purpose of this feature.